src/Tests/dotnet.Tests/ParserTests/ToolRunParserTests.cs

```diff
@@ -37,7 +35,6 @@ public void ListToolParserCanGetCommandsArgumentInUnmatchedTokens()
             var result = Parser.Instance.Parse("dotnet tool run dotnetsay hi");
             result.UnmatchedTokens.Should().Contain("hi");
             result.UnparsedTokens.Should().BeEmpty();
-            result.Errors.Should().BeEmpty();
```

Comment:
It has errors now? What are they?

Reply:
System.CommandLine reports any errors if there are unmatched tokens. So `dotnet tool run dotnetsay -- hi` is fine but `dotnet tool run dotnetsay hi` causes an error.

Reply:
similar to the other comment. it will still run right? We disabled the error when it is unmatched right?

Reply:
Yes, we have `TreatUnmatchedTokensAsErrors` set to false for both the run commands